config/routes.rb

```diff
@@ -1,11 +1,12 @@
 Rails.application.routes.draw do
 
+root "static_pages#home"
   scope "(:locale)", locale: /en|vi/ do
-    get "help" => "static_pages#help"
-    get "about" => "static_pages#about"
-    get "contact" => "static_pages#contact"
-  resource: users
-    get "signup" => "users#new"
-    root "static_pages#home"
+    get "/help" => "static_pages#help"
```

Comment:
em viết lại dòng này và những dòng khác theo cú pháp như này nha:
```suggestion
    get "/help", to: "static_pages#help"
```